### DISCUSSION TITLE
Qwen-TensorRT-LLM 编译 QWen-7b-chat-int4，推理的时候，结果全是感叹号！

### DISCUSSION THREAD

**Discussion by samir1224** [Tlntin/Qwen-TensorRT-LLM#89] [Q&A]
<img width="728" alt="image" src="https://github.com/Tlntin/Qwen-TensorRT-LLM/assets/3377090/f0b29279-0bb1-43ea-a415-223d2224d8d5">
请问怎么回事呢?

**Comment by Tlntin** [comment]
升级下transformers版本到最新版试试
